core/src/main/java/io/parsingdata/metal/expression/value/reference/Nth.java

```diff
@@ -46,9 +46,6 @@
  * <code>values</code>. For every invalid index (such as
  * {@link Optional#empty()}, a negative value or an index that is out of
  * bounds) empty is returned.
- *
- * @see NameRef
- * @see TokenRef
```

Comment:
Some of those `@see`s were quite random. The only one that made sense was a reference from `token/TokenRef` because the name fully overlapped. But now that it doesn't overlap anymore, I've removed it as well.